### PR TITLE
Updated bootstrap styling including addition of admin logout.

### PR DESCRIPTION
**Pull request by mvillis**
**Some minor tweaks to presentation of the site.**

* Login / logout now in top right hand corner (see preview screenshot below)
* Added space between forms and the submit buttons (using form-group classes)
* Moved about to the right hand side next to the login / logout option to leave space for future menu

**Previews:**
New home screen (note the login / register buttons will be replaced by a link to the admin screen if logged in)
home-screen

Option to logout now appears as a drop down in the top right
logout-option

Smoothed user feedback form
screen shot 2015-01-05 at 8 41 19 pm

### CODE REVIEW

File: teamtemp/templates/about.html [hunk 1]
```diff
@@ -1,27 +1,30 @@
 {% extends "base.html" %}
 
-{% block title %}About Team Temperature{% endblock %}
+{% block title %}About Team Temp{% endblock %}
 
 {% block body %}
-
+<div class="container">
```
Comment (travisjwarren) [inline teamtemp/templates/about.html]: Should this be in the base template?

All body content will be in a container, will be end up with lots of duplicate code by adding a container to each template?

File: teamtemp/templates/base.html [hunk 2]
```diff
@@ -4,40 +4,46 @@
...
-    </div>
-    {% block javascript %}
-        <script src="https://code.jquery.com/jquery.js"></script>
-        <script src="{% static "js/bootstrap.min.js" %}">
-    {% endblock %}
-</body>
-</html>
+    <!-- /.container-fluid -->
+</nav>
+{% block body %}{% endblock %}
```
Comment (travisjwarren) [inline teamtemp/templates/base.html]: See previous comment.  Wrap in container div?

File: teamtemp/templates/about.html (hunk 1 above)
Comment (mvillis) [inline teamtemp/templates/about.html]: Wasn't sure if I wanted to force every page to use a container or whether there would be a scenario where the body was inside a container-fluid as opposed to a container. At the moment we don't have that scenario.

Happy to extract out the container declaration into the base and should the moment come for us ever to need a fluid one we can refactor.

Will submit some changes shortly.

**Comment by mvillis** [comment]
did a pre-emptive squash commit @travisjwarren
changes are ready for review and possible :ship:

**Pull request merged**